test/units/module_utils/network/common/test_parsing.py

```diff
@@ -0,0 +1,23 @@
+from ansible.compat.tests import unittest
```

Comment:
Please add prefix like https://github.com/ansible/ansible/blob/devel/test/units/module_utils/network/common/test_utils.py#L1-L22 related to license and code practice on the top.